docs/ref/clvm.md

```diff
@@ -125,24 +125,23 @@ And in this example, an atom that was read in as an integer is appended to a str
 
 ### Atoms as Byte Arrays
 
-The atom is treated as an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
+The atom is an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
 
-### Unsigned Integer
+### Signed Integer
 
-An unsigned integer of arbitrary length. If more bits are required to perform an operation with atoms of different length, the atom is virtually extended with zero bytes to the left.
+Arithmetic operations will interpret its operands as two's complement, big endian, signed integers. The most significant bit denotes a negative number. In order to represent a positive integer where the most significant bit aligns with the most significant bit in the first byte need a 0-byte prefix in order to be interpreted as positive.
 
-### Signed Integer
+Said another way, if a positive integer's first byte is >= `0x80` then it will be prepended with a `0x00`. Without that prepended byte, a positive value would appear negative in the case that the high bit is set.
 
-The byte array behaves as a two's complement signed integer. The most significant bit denotes a negative number. The underlying representation matters, because the individual bytes are viewable through other operations.
+e.g. `0xFF` means -1 and `0x00FF` means 255.
 
-This type has the potential for multiple representations to be treated as the same value. For example, 0xFF and 0xFFFF both encode `-1`. Integer arithmetic operations that treat returned atoms as signed integers will return the minimal representation for negative numbers, eg. `0xFF` for `-1`
```

Comment:
In "0xFF and 0xFFFF both encode `-1`" I think "represent" is clearer than "encode".

In "Integer arithmetic operations". I think "Integer" is redundant.

I think "shortest representation" is clearer than "minimal" (since "minimal" can also relate to values).